modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/enricher/JavaOAuthDeclarationEnricher.java

```diff
@@ -170,16 +187,20 @@ private OperationDeclaration buildUnauthorizeOperation() {
       operation
           .addModelProperty(new CompletableComponentExecutorModelProperty((model, params) -> new UnauthorizeOperationExecutor()));
 
-      ParameterGroupDeclaration group = operation.getParameterGroup(DEFAULT_GROUP_NAME);
-      group.showInDsl(false);
-      ParameterDeclaration parameter = new ParameterDeclaration(RESOURCE_OWNER_ID_PARAMETER_NAME);
-      parameter.setDescription("The id of the resource owner which access should be invalidated");
-      parameter.setExpressionSupport(SUPPORTED);
-      parameter.setLayoutModel(LayoutModel.builder().build());
-      parameter.setRequired(false);
-      parameter.setParameterRole(BEHAVIOUR);
-      parameter.setType(stringType, false);
-      group.addParameter(parameter);
+      if (supportsAuthCode) {
+        ParameterGroupDeclaration group = operation.getParameterGroup(DEFAULT_GROUP_NAME);
+        group.showInDsl(false);
```

Comment:
? why is it doing this to the default group?

Reply:
client credentials doesn't have the concept of resourceOwnerId. AUth code is the only grant type that supports the concept of multitenancy